### DISCUSSION TITLE
Questionnaire sur des "non-cyclistes"

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#3] [Q&A] [answered]
@VeloTerritoires
Lors de l'analyse du fichier "modèle", j'ai trouvé 148 questionnaires qui semblent avoir été rempli par des piétons:
exemples : 38A43, 38A44
Certains de ces piétons ont même répondu à l'interview ET au questionnaire (exemple: 4A2, 4A3)

Est-ce que je dois:

- Ignorer les réponses aux questionnaires et à l'interview pour les "non-cyclistes"
- Considérer qu'il y a eu une erreur dans [categorie_visuelle] et les considérer comme des cyclistes

**Comment by SteffMangin** [comment, marked as answer]
L'enquête en Bretagne a été administrée aussi à des piétons. J'aurai du te les exclure de la base et j'ai oublié. A ce stade de développement de la méthode, tu peux les ignorer.
